BaseInterfaces/test/runtests.jl

```diff
@@ -6,6 +6,22 @@ using Test
 @implements SetInterface{(:empty,:emptymutable,:hasfastin,:intersect,:union,:sizehint!)} Test.GenericSet [Test.GenericSet(Set((1, 2)))]
 @implements DictInterface Test.GenericDict [Arguments(d=Test.GenericDict(Dict(:a => 1, :b => 2)), k=:c, v=3)]
 
+@testset "inheritance" begin
+    # Inherited interfaces are stored in the second parameter of the supertype
+    @test supertype(ArrayInterface) <: Interfaces.Interface{<:Any,IterationInterface{(:reverse, :indexing)}}
+    @test supertype(DictInterface) <: Interfaces.Interface{<:Any,IterationInterface{(:reverse,)}}
```

Comment:
```suggestion
    @test supertype(ArrayInterface) <: Interfaces.Interface{<:Any,>:IterationInterface{(:reverse, :indexing)}}
    @test supertype(DictInterface) <: Interfaces.Interface{<:Any,>:IterationInterface{(:reverse,)}}
```

Reply:
in case we define more interfaces in the future and the second parameter is a union, this seems more robust?

Reply:
also why not just test `ArrayInterface <: ...`?